src/types/Account/methods/createTransaction.spec.js

```diff
@@ -0,0 +1,191 @@
+const {expect} = require('chai');
+const {HDPrivateKey} = require('@dashevo/dashcore-lib');
+
+const createTransaction = require('./createTransaction');
+const mnemonic = require('../../../../fixtures/wallets/mnemonics/during-develop-before').mnemonic;
+const FakeNet = require('../../../../fixtures/FakeNet/FakeNet');
+
+const _ = require('lodash');
+// const createTransaction = require('./createTransaction');
+const getUTXOS = require('./getUTXOS');
+const {simpleDescendingAccumulator} = require('../../../utils/coinSelections/strategies');
+
+const addressesFixtures = require('../../../../fixtures/addresses.json');
+const validStore = require('../../../../fixtures/walletStore').valid.orange.store;
+
+const craftedGenerousMinerStrategy = (utxosList, outputsList, deductFee = false, feeCategory = 'normal') => {
```

Comment:
don't you want to move it to a separate file?

Reply:
I should have wanted it as I guess that's why there is a fixtures folder >.>" 
Doing it rn!